tests/testthat/test_XcmsExperiment.R

```diff
@@ -1326,3 +1326,34 @@ test_that("storeResults,RDataParam works", {
     expect_s4_class(object, "XcmsExperiment")
     expect_equal(object, xmse)
 })
+
+test_that("storeResults,PlainTextParam,MsExperiment works", {
+    param <- PlainTextParam(path = "/path/test/")
+    param2 <- PlainTextParam()
+    expect_false(is.null(param2))
+    # Test if PlainTextParam is invalid with multiple paths
+    expect_error(new("PlainTextParam", path = c(tempdir(), tempdir())))
+    mse <- filterRt(mse, c(100,150)) #this adds to processing but not processingQueue
+    storeResults(mse, param = param)
+    expect_true(dir.exists("/path/test/"))
+    expect_true(file.exists(file.path(param@path, "sample_data.txt")))
+    expect_true(file.exists(file.path(param@path, "spectra_files.txt")))
+    expect_true(file.exists(file.path(param@path, "spectra_processing_queue.json"))) # empty so does not work
```

Comment:
A `filterMzRange` call will show up in the processing queue (while `filterRt` will not) - so, with that this file should actually be present then.